devito/core/__init__.py

```diff
@@ -32,6 +34,10 @@
 operator_registry.add(DeviceOpenMPOperator, Device, 'advanced')
 operator_registry.add(DeviceOpenMPCustomOperator, Device, 'custom')
 
+operator_registry.add(DeviceOpenACCNoopOperator, Device, 'noop', 'openacc')
+operator_registry.add(DeviceOpenACCOperator, Device, 'advanced', 'openacc')
+operator_registry.add(DeviceOpenACCCustomOperator, Device, 'custom', 'openacc')
+
```

Comment:
So shouldn't the rest of the registery have a language too like openmp for OMPoffloading and C for the rest, only ths one seems to use language

Reply:
yes. `operator_registry` defaults to `language=C` if not provided. Currently, we still have the issue that OpenMP is somewhat special (it even has its own env var and compilation passes!).

Some cleanup is necessary, but I'd say that's for another day